### PR TITLE
Add duty supervisor to database and website (and support for /whoson)

### PR DESCRIPTION
**Pull request by tceluzza**
This adds a `dutysup` row to the `crews` table and the `default_crews` table in the database. It also adjusts the `Modify Schedule` and `Edit Default Schedule` pages on the website to show a column for Duty Supervisor.

Since `js/controllers/NightCrewsCtrl.js` and `js/controllers/ModifyScheduleCtrl.js` both used the same `loadCrews()` function to display the crew, the DS was appearing on the Night Crews page, I added a second function `loadCrewsNoDS` that removes the DS column from the Night Crews Page. Alternatively, it could be removed when being displayed, leaving it in the array, which would only require one function. Let me know if you think that'd be better!

### CODE REVIEW

File: .crews.php [hunk 1]
```diff
@@ -315,7 +316,7 @@ function main () {
                 }
 
                 $positions = [
-                    'cc', 'driver', 'attendant', 'observer'
+                    'cc', 'driver', 'attendant', 'observer', 'dutysup'
```
[the remark this answers is not in the document]
Comment (ddbruce) [reply .crews.php:319]: Yes, that is the correct logic. The DS does not count towards any night crew personnel requirements

File: js/controllers/ModifyScheduleCtrl.js [hunk 2]
```diff
@@ -48,6 +48,35 @@ angular.module(ctrl_name, []).controller(ctrl_name, ['$scope', '$http', '$q', 'A
...
+                // them in a way that almost seems backwards.
+                $scope.tableHeadings = response.data.headings;
+                $scope.crews = response.data.crews;
+                $scope.positions = response.data.positions;
+                $scope.loadedCrews = true;
+            }
+        });
+    };
+
+
```
[the remark this answers is not in the document]
Comment (ddbruce) [reply js/controllers/ModifyScheduleCtrl.js:79]: Generally, we should be in the habit of only writing code once. This website is probably not the shining example of that, but any modifications/upgrades/whatever we do should try to stick to that philosophy.

**Comment by tceluzza** [comment]
I think I've fixed everything - uses one one `loadCrews` function, doesn't check duty sups when checking if a D-T/CC-T is on,  and allows a member to be set as both duty sup and another position in the Modify Schedule page on the website.

File: .crews.php [hunk 3]
```diff
@@ -315,16 +316,21 @@ function main () {
...
                 // Loops through all the spots and checks if there is a CC on in any of them.
                 foreach($positions as $pos) {
+                    // Duty sup is not actively on night crew
+                    if($pos == 'dutysup') {
+                        continue;
+                    }
+
                     $statement = $connection->prepare("SELECT * FROM members WHERE id = :memberid LIMIT 1");
-                    $statement->bindValue(':memberid', $y['cc']);
+                    $statement->bindValue(':memberid', $y[$pos]);
```
Comment (tceluzza) [inline .crews.php:330]: This was only checking the CC crew slot, even though it was looping through each position

**Review by ddbruce — APPROVED** [review]
lgtm, let 'er rip kid

**Pull request merged**